velox/common/base/BitUtil.cpp

```diff
@@ -26,8 +26,8 @@ namespace facebook::velox::bits {
 namespace {
 // Naive implementation that does not rely on BMI2.
 void scatterBitsSimple(
-    int32_t numSource,
-    int32_t numTarget,
+    uint32_t numSource,
```

Comment:
This particular change could cause problem when `numTarget` is 0.  And it won't improve performance (only when we do division on unsigned number we will see perf gain).  We should leave them as signed.